src/vs/base/browser/ui/breadcrumbs/breadcrumbsWidget.ts

```diff
@@ -340,6 +344,10 @@ export class BreadcrumbsWidget {
 		container.tabIndex = -1;
 		container.setAttribute('role', 'listitem');
 		container.classList.add('monaco-breadcrumb-item');
+		if (item.supportDrag()) {
+			container.draggable = true;
+			this._disposables.add(dom.addStandardDisposableListener(container, 'dragstart', e => this._onCrumbDrag(e)));
```

Comment:
This is not OK. Everytime an item is rendered (which happens repeatedly) a listener is added but _not_ disposed unless the breadcrumbs widget is disposed. The listeners must be managed with the lifecycle of the _renderItem-call. 

Because of this and to avoid having too many listeners I recommended doing this:

* only set draggable here
* have only a single drag listener for the root dom node
* have logic in the single listener to figure out what item is being dragged